cluster/gce/gci/configure.sh

```diff
@@ -126,7 +134,7 @@ function validate-hash {
   local -r file="$1"
   local -r expected="$2"
 
-  actual=$(sha1sum ${file} | awk '{ print $1 }') || true
+  actual=$(sha1sum "${file}" | awk '{ print $1 }') || true
```

Comment:
You can avoid the dependency towards awk here by using built-in shell script functionality (takes 2 lines):

actual=$(sha1sum "${file}")
actual=${actual%%[[:blank:]]*}